modular_skyrat/master_files/code/modules/jobs/job_types/roboticist.dm

```diff
@@ -4,10 +4,21 @@
 	duffelbag = /obj/item/storage/backpack/duffelbag/science/robo
 	messenger = /obj/item/storage/backpack/messenger/science/robo
 
+	glasses = /obj/item/clothing/glasses/hud/diagnostic
+	gloves = /obj/item/clothing/gloves/color/black
+
+	l_hand = /obj/item/storage/medkit/mechanical/roboticist
+
 /datum/job/roboticist
 	description = "Build cyborgs, mechs, AIs, and maintain them all. Create MODsuits for those that wish. Try to remind medical that you're \
 	actually a lot better at treating synthetic crew members than them."
 
+/datum/outfit/job/roboticist/New()
+	. = ..()
+
+	LAZYINITLIST(backpack_contents)
+	backpack_contents[/obj/item/clothing/head/utility/welding] = 1
```

Comment:
welding helmets are really important for treatment

Reply:
you cant use a welding tool without it, and im trying to maintain some level of parity between MDs and roboticists

if a MD can come off the shuttle and immediately start suturing someone, a robo should be able too as well